components/stacks-network/src/orchestrator.rs

```diff
@@ -1343,7 +1343,7 @@ observer_port = {subnet_events_ingestion_port}
             stacks_node_p2p_port = devnet_config.stacks_node_p2p_port,
             orchestrator_port = devnet_config.orchestrator_ingestion_port,
             subnet_events_ingestion_port = devnet_config.subnet_events_ingestion_port,
-            first_burn_header_height = 0,
+            first_burn_header_height = 0, // todo: I don't think this should be 0
```

Comment:
I removed this comment.
- This for the subnet config so I think we can ignore it for now
- I don't find references to this config in the stacks-repo so maybe we'll fully remove it in the future